web/viewer.css

```diff
@@ -778,9 +803,13 @@ select {
   mask-image: var(--toolbarButton-menuArrow-icon);
 }
 
+#scaleSelectContainer {
+  border: 1px solid ButtonText;
+}
+
```

Comment:
This will affect the "normal", i.e. non-HCM, mode as well which I'm not sure is actually intended here!?

- If not intended, then you'll want to add a new `--scale-select-container-border: none` variable at the top of the file and a `border: var(--scale-select-container-border);` line to the `.dropdownToolbarButton`-class just above. Finally, for the `forced-colors: active`-case, you'll obviously add the value used here.

 - If actually intended, then we'll definitely need to tweak a lot of the existing margin/padding-values to avoid this element shifting and growing compared to its current state. Also, I'm guessing that the color-value shouldn't just be `ButtonText` in "normal" mode since that looks quite out-of-place in the toolbar?